Sources/PointFree/Database.swift

```diff
@@ -386,6 +482,15 @@ private func migrate() -> EitherIO<Error, Prelude.Unit> {
       )
       """
     )))
+    .flatMap(const(execute(
+      """
+      CREATE TABLE IF NOT EXISTS "email_settings" (
+        "id" uuid DEFAULT uuid_generate_v1mc() PRIMARY KEY NOT NULL,
+        "newsletter" character varying,
```

Comment:
not using a postgres enum cause we dont really have a nice way to keep it in sync with our swift enum. but open to better ways of doing this. 

luckily our router should completely prevent invalid values from showing up in here